dune-project

```diff
@@ -1 +1 @@
-(lang dune 1.0)
```

Comment:
why is that needed? :)

Reply:
The new rules I wrote use some `dune.1.11` features such as the `target` field. I can try to lower the required version down a bit if you want!

Reply:
were the old rules wrong?

Reply:
Sorry, I forgot to answer! Yeah using a `progn` in this case isn't very friendly to dune as it doesn't know about generated files. Also the `diff?` behaviour isn't the best, we've had issue with these mdx rules in the past.